### PR TITLE
Issue 54 calorie calculator

### PR DESCRIPTION
**Pull request by rogercyyu**
Hello, this adds a calorie counter to close [issue-54](https://github.com/drodil/rustybeer/issues/54). 
Couples things I would like to mention:
- Hard to calculate ABV using one value, so I decided to use JSON to lookup the range for ABV 1.0 to 22.0. Chart [here].(http://simplybeer.com/how-many-calories-are-in-my-beer/)
- The equations to calculate calories for beer are for 12 oz. I decided to convert them over to ml. (not sure if accuracy is affected) 
- Not sure if there is a standard for how you would like things outputted.

Anyways, review and feedback is greatly appreciated. Thanks for letting me try this! :)

### CODE REVIEW

**Review by mlatief — CHANGES_REQUESTED** [review]
Looks good, I suggest using `rustybeer_util::conversions::MassBuilder` for `oz` to `ml` conversion, and resolving conflicts before being able to merge.

File: rustybeer-cli/src/commands/calories.rs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    }
+}
+
+pub fn do_matches<'a>(matches: &ArgMatches<'a>) {
+    if let Some(matches) = matches.subcommand_matches("calories") {
+        if matches.is_present("fg") && matches.is_present("og") {
+            let fg = value_t!(matches, "fg", f32).unwrap_or_else(|e| e.exit());
+            let og = value_t!(matches, "og", f32).unwrap_or_else(|e| e.exit());
+            let cv = value_t!(matches, "volume", String).unwrap_or_else(|e| e.exit());
+            let conversion = MassBuilder::from_str("12oz").unwrap().as_grams() as f32;
```
Comment (drodil) [inline rustybeer-cli/src/commands/calories.rs:56]: You should use the volumes of different bottles merged in #54 and show calories for each volume

Comment (rogercyyu) [reply rustybeer-cli/src/commands/calories.rs:56]: Hi, to clarify. What you want is a table that list out all of the calories for each volume provided by `num_bottles` but if the user also inputs a volume, then it should only output the calories for that particular volume? So they will either get a table of defaults or just a breakdown for their particular volume.

Comment (drodil) [reply rustybeer-cli/src/commands/calories.rs:56]: So basically calorie count for 1 of each bottle type + optional custom volume

Comment (rogercyyu) [reply rustybeer-cli/src/commands/calories.rs]: When just providing og and fg:
image

When providing og, fg, and volume:
image

When providing abv:
image

When providing abv and volume:
image

I hope this works.

**Review by mlatief — APPROVED** [review]
This looks good now :+1:

**Comment by mlatief** [comment]
@all-contributors please add @rogercyyu for Code

**Pull request merged**